test/contract/sequencerInboxDelayBufferable.spec.ts

```diff
@@ -477,19 +481,21 @@ describe('SequencerInboxDelayBufferable', async () => {
 
     const txn = sequencerInbox
       .connect(batchPoster)
-    [
-      'addSequencerL2BatchFromOrigin(uint256,bytes,uint256,address,uint256,uint256,bytes)'
-    ](
-      2,
-      data,
-      delayedMessageCount.add(2),
-      ethers.constants.AddressZero,
-      seqReportedMessageSubCount.add(20),
-      seqReportedMessageSubCount.add(30),
-      '0x',
-      { gasLimit: 10000000 }
+      [
+        'addSequencerL2BatchFromOrigin(uint256,bytes,uint256,address,uint256,uint256)'
```

Comment:
Why are we not calling the espresso function here? Is it because we are not enabling delay proofs?